paddle/fluid/framework/operator.h

```diff
@@ -310,6 +310,18 @@ class ExecutionContext {
   const RuntimeContext& ctx_;
 };
 
+inline bool CanCUDNNBeUsed(const framework::ExecutionContext& ctx) {
```

Comment:
CanUseCudnn or CanUseCUDNN?

Reply:
Maybe it is not important. Similar name appears such as `CanMKLDNNBeUsed`. I just move this function from `cudnn_helper.h` to `operator.h`, because it accepts `framework::ExecutionContext` as its parameter. It is better to put it inside `operator.h`.